charts/ai-software-templates/chatbot/0.1.0/Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v2
-description: This is an example chart for a chatbot application based on the AI RHDH Software template approach. For more information please check https://github.com/redhat-ai-dev/ai-lab-template.git
+description: A helm chart for the AI Software Templates. For more information please check https://github.com/redhat-ai-dev/ai-lab-helm-charts.git
 name: chatbot-ai-software-template
-tags: chatbot,python,llama.cpp,vllm
+tags: chatbot,llama.cpp,ai-lab-template
```

```suggestion
tags: chatbot,llama.cpp,ai-template
```

I'm going back and forth on if I like `ai-lab-template` or `ai-template` better for the tag. wdyt?

---

Me too, the reason I'd prefer ai-lab-template is just to have a connection with the current name of the template repo.

---

What about just `ai-lab`?

I know we want a connection with the template repo, but does that tag make sense in the context of the Helm chart?

---

I think `ai-lab` makes sense too, keeps the "branding" we have for our ai projects. I've updated the tags